docker-compose.yml

```
@@ -74,66 +12,12 @@
-        condition: service_healthy
-      redis:
-        condition: service_healthy
-    environment:
-      - POSTGRES_HOST=db
-      - REDIS_HOST=redis
-
-volumes:
-  postgres_data:
-  redis_data:
-  minio_data:
+      - CODESPACES=true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)